README.md

```diff
@@ -74,6 +74,8 @@ Download model's weights from [Dropbox](https://www.dropbox.com/s/t7h24l6wx9vret
 #### Linux
 Linux uses `v4l2loopback` to create virtual camera.
 
+### Native installation
```

Comment:
I would leave Linux section as is and add another section called Docker on the same level, like
### Linux
...
### Docker